tests/kohana/RequestTest.php

```diff
@@ -730,9 +722,9 @@ public function test_passes_client_params()
 
 } // End Kohana_RequestTest
 
-class Controller_Kohana_RequestTest_Dummy extends Controller
+class Controller_Kohana_RequestdTest_Dummy extends Controller
```

Comment:
This is really strange.

Now that I am reviewing this myself, I wonder how the tests passed with the "d" in the middle of `RequestdTest`. I remember I was playing around when I added a letter, in order to check if the routes were picking the controller and the tests were passing/failing properly.

Not sure how it got committed and why the tests are green.